src/Home/Home.css

```diff
@@ -0,0 +1,9 @@
+.home {
+  border: 2px solid #e4afff;
```

Comment:
I might be able to get rid of the border, or change it's styling if I had more time. I will leave it for now.